pygac_fdr/tests/test_end2end.py

```diff
@@ -0,0 +1,470 @@
+            'midnight_line': np.nan,
+            'overlap_free_end': 10770,
+            'global_quality_flag': QualityFlags.OK
+        },
+        'NSS.GHRR.NA.D81089.S1459.E1631.B0912829.GC': {
+            'nc_file': 'AVHRR-GAC_FDR_1C_N06_19810330T145914Z_19810330T163142Z_R_O_20200101T000000Z_0100.nc',
+            'midnight_line': np.nan,
+            'overlap_free_end': 10489,
+            'global_quality_flag': QualityFlags.OK
+        },
+        'NSS.GHRR.NA.D81089.S1626.E1812.B0912930.GC': {
+            'nc_file': 'AVHRR-GAC_FDR_1C_N06_19810330T162639Z_19810330T181201Z_R_O_20200101T000000Z_0100.nc',
+            'midnight_line': np.nan,
+            'overlap_free_end': 12057,
+            'global_quality_flag': QualityFlags.OK
+        },
+        'NSS.GHRR.NA.D81089.S1807.E1951.B0913031.GC': {
+            'nc_file': 'AVHRR-GAC_FDR_1C_N06_19810330T180708Z_19810330T195132Z_R_O_20200101T000000Z_0100.nc',
+            'midnight_line': np.nan,
+            'overlap_free_end': 12011,
+            'global_quality_flag': QualityFlags.OK
+        },
+        'NSS.GHRR.NA.D81089.S1947.E2131.B0913132.GC': {
+            'nc_file': 'AVHRR-GAC_FDR_1C_N06_19810330T194718Z_19810330T213158Z_R_O_20200101T000000Z_0100.nc',
+            'midnight_line': np.nan,
+            'overlap_free_end': 11955,
+            'global_quality_flag': QualityFlags.OK
+        },
+        'NSS.GHRR.NA.D81089.S2126.E2256.B0913233.WI': {
+            'nc_file': 'AVHRR-GAC_FDR_1C_N06_19810330T212656Z_19810330T225612Z_R_O_20200101T000000Z_0100.nc',
+            'midnight_line': np.nan,
+            'overlap_free_end': 10057,
+            'global_quality_flag': QualityFlags.OK
+        },
+        'NSS.GHRR.NA.D81089.S2251.E0035.B0913334.WI': {
+            'nc_file': 'AVHRR-GAC_FDR_1C_N06_19810330T225108Z_19810331T003506Z_R_O_20200101T000000Z_0100.nc',
+            'midnight_line': 8260,
+            'overlap_free_end': 12472,
+            # This is the last file here, therefore overlap_free_end == along_track - 1.
+            # In CLARA there is overlap with the next file, which is why they have 12101.
+            'global_quality_flag': QualityFlags.OK
+        },
+    }  # from CLARA-A3 Feedback Loop 1
+
+    @classmethod
+    def setUpClass(cls):
+        super(EndToEndTestNormal, cls).setUpClass()
+
+        gac_files_gz = [os.path.join(cls.input_dir, 'normal', fname + '.gz')
+                        for fname in cls.mda_exp.keys()]
+        cls.nc_files_ref = [os.path.join(cls.output_dir_ref, 'normal', mda['nc_file'])
+                            for mda in cls.mda_exp.values()]
+        dbfile = os.path.join(cls.test_data_dir, 'test.sqlite3')
+        if cls.fast:
+            gac_files_gz = [gac_files_gz[-1]]
+            cls.nc_files_ref = [cls.nc_files_ref[-1]]
+
+        # Process "normal" files
+        cls.output_dir = os.path.join(cls.test_data_dir, 'output', 'normal')
+        cls.nc_files = cls._run(gac_files_gz, dbfile)
```

Comment:
This can be a separate method (and can be shared with the next class if you pass `normal` or `corrupt` as argument

Reply:
Good point, will do!